tachyon/zk/plonk/permutation/permutation_assembly.h

```diff
@@ -80,10 +80,10 @@ class PermutationAssembly {
 
   // Returns |PermutationVerifyingKey| which has commitments for permutations.
   constexpr PermutationVerifyingKey<PCS> BuildVerifyingKey(
-      const Entity<PCS>* entity, const std::vector<Evals>& permutations) const {
```

Comment:
```suggestion
      const Entity<PCS>* entity, const std::vector<Evals>& permutations) const {
```

Reply:
@chokobole I can't see the difference for the suggestion. Can you check again please?

Reply:
Oh you meant to fix back to `const Entity<PCS>* entity`. I'll just put the const for the `BuildVerifyingKey` function and remove non-const getter `permutations`!